src/lib-components/BlockTag.vue

```diff
@@ -61,13 +64,24 @@ const parsedClasses = computed(() => {
 
 <template>
   <span :class="parsedClasses">
```

Comment:
Rename parsedClasses to classes, like other components toplevel computed class method